srm_oparray.c

```diff
@@ -110,7 +124,11 @@ static const op_usage opcodes[] = {
 #else
 	/*  49 */	{ "CHECK_VAR", OP1_USED },
 	/*  50 */	{ "SEND_VAR_NO_REF_EX", ALL_USED },
-	/*  51 */	{ "MAKE_REF", RES_USED| OP1_USED },
+#if PHP_VERSION_ID >= 70400
+	/*  51 */	{ "CAST", ALL_USED | EXT_VAL },
+#else
+	/*	51 */	{ "MAKE_REF", RES_USED| OP1_USED},
```

Comment:
Can you fix the white space here please?

Reply:
OK. I will fix it soon.

Reply:
I am sorry that I used tab here.  And I have fix it by replacing tab with white spaces. Please review it again.